frontend/package.json

```
@@ -32,17 +32,24 @@
     "vue-router": "^3.0.1",
     "vuetify": "^2.1.0",
     "vuex": "^3.0.1",
+    "vuex-module-decorators": "itggot-TE4/vuex-module-decorators",
     "vuex-persist": "^2.2.0",
-    "vuex-persistedstate": "^2.7.0",
-    "vuex-module-decorators": "itggot-TE4/vuex-module-decorators"
+    "vuex-persistedstate": "^2.7.0"
```

```suggestion
    "vuex-persistedstate": "^2.7.0"
```
This package isn't used